pkg/virtblock/virtblock.go

```diff
@@ -0,0 +1,13 @@
+package virtblock
+
+import (
+	"fmt"
+
+	"github.com/virtblocks/virtblocks/go/rust/pkg/devices"
+)
+
+func CreateMemBalloon() error {
+	balloon := devices.NewMemballoon()
```

Comment:
The Go → Rust bindings for Virt Blocks require objects to be free'd manually, so there should be a `defer balloon.Free()` right after this line.